nuget.test/tests/package_content.Tests.ps1

```diff
@@ -7,38 +7,51 @@ Import-Module (Join-Path $modulesPath global_config.psm1) -Force
 		$agentVersion = "3.11.283.0"
 		
 		Context "When package exists" {
-			It "checks to see if content\newrelic.config exists" {
-				(Test-Path $PackageRoot\content\newrelic\newrelic.config) | Should Be $true
-			}
-			
-			It "checks to see if content\newrelic.xsd exists" {
-				(Test-Path $PackageRoot\content\newrelic\newrelic.config) | Should be $true
-			}
-			
-			It "checks to see if content\newrelic\NewRelic.Agent.Core.dll exists" {
-				(Test-Path $PackageRoot\content\newrelic\NewRelic.Agent.Core.dll) | Should Be $true
-			}
-			
-			It "checks to see if content\newrelic\NewRelic.Agent.Core.pdb does not exist" {
-				(Test-Path $PackageRoot\content\newrelic\NewRelic.Agent.Core.pdb) | Should be $false
+			$newRelicHome = "$PackageRoot\content\newrelic";
+			$expectedFiles = @(
+				"$newRelicHome\newrelic.config",
+				"$newRelicHome\NewRelic.Agent.Core.dll",
+				"$newRelicHome\NewRelic.Profiler.dll",
+				"$newRelicHome\NewRelic.Agent.Extensions.dll",
+				"$newRelicHome\newrelic.xsd",
+				"$newRelicHome\extensions\CoreInstrumentation.xml",
+				"$newRelicHome\extensions\extension.xsd",
+				"$newRelicHome\extensions\NewRelic.Providers.TransactionContext.Asp.dll",
+				"$newRelicHome\extensions\NewRelic.Providers.TransactionContext.Default.dll",
+				"$newRelicHome\extensions\NewRelic.Providers.TransactionContext.Wcf3.dll",
+				"$newRelicHome\extensions\NewRelic.Providers.Wrapper.NServiceBus.dll",
+				"$newRelicHome\extensions\NewRelic.Providers.Wrapper.NServiceBus.Instrumentation.xml"

```

Comment:
In the future we might want to abstract this out to a separate file (assuming the list may grow) just to keep the noise down in the test.